README.md

```diff
@@ -2,25 +2,35 @@
 
 [![Circle CI](https://circleci.com/gh/Financial-Times/topics-transformer/tree/master.png?style=shield)](https://circleci.com/gh/Financial-Times/topics-transformer/tree/master)
 
-Retrieves Topics taxonomy from TME vie the structure service and transforms the topics to the internal UP json model.
+Retrieves Topics taxonomy from TME and transforms the topics to the internal UP json model.
 The service exposes endpoints for getting all the topics and for getting topic by uuid.
 
-# Usage
+# Hot to run
 `go get github.com/Financial-Times/topics-transformer`
```

Comment:
You are looking at outdated code or readme ... Dafydd already pointed out  my atrocious spelling and grammar and it's been corrected